src/main/java/org/zalando/nakadi/controller/CursorsController.java

```diff
@@ -83,7 +82,7 @@ public CursorsController(final CursorsService cursorsService,
         try {
             final List<SubscriptionCursor> cursors = cursorsService.getSubscriptionCursors(subscriptionId, client)
                     .stream()
-                    .map(cursor -> cursorConverter.convert(cursor, cursorTokenService.generateToken()))
+                    .map(cursor -> cursor.withToken(cursorTokenService.generateToken()))
```

Comment:
@adyach To have fancy hipster code (well, it's quite handy to have this method, and not to use ctor)